aten/src/ATen/native/vulkan/VulkanGuardImpl.cpp

```diff
@@ -25,14 +28,17 @@ struct VulkanGuardImpl final : public c10::impl::DeviceGuardImplInterface {
     // no-op
   }
   void uncheckedSetDevice(Device d) const noexcept override {
+    (void)d;
```

Comment:
Since we are in C++17, you could also use `maybe_unused` https://en.cppreference.com/w/cpp/language/attributes/maybe_unused